### ISSUE TITLE
binary command not created into /bin as explain on the doc 

### ISSUE THREAD

**Issue by jeanmonod** [rokka-io/rokka-client-php-cli#13]
In the readme, it says:

    run bin/rokka-cli.php

But after a fresh install, there is not bin folder

```
vagrant@zebra:/vagrant$ mkdir rokka-cli && cd rokka-cli
vagrant@zebra:/vagrant/rokka-cli$ composer require rokka/client-cli
...
symfony/dependency-injection suggests installing symfony/expression-language (For using expressions in service container configuration)
symfony/dependency-injection suggests installing symfony/proxy-manager-bridge (Generate service proxies to lazy load them)
symfony/console suggests installing symfony/event-dispatcher ()
symfony/console suggests installing symfony/process ()
Writing lock file
Generating autoload files
vagrant@zebra:/vagrant/rokka-cli$ ll
total 36K
-rw-r--r-- 1 501 dialout  62 Nov 18 11:58 composer.json
-rw-r--r-- 1 501 dialout 28K Nov 18 11:58 composer.lock
drwxr-xr-x 9 501 dialout 306 Nov 18 11:58 vendor
```

**Comment by thePanz** [comment]
The documentation needs some love :)

To use the CLI via `composer require` the rokka-cli binary file is in your `vendor/bin` folder as `vendor/bin/rokka-cli`.
If you want to use a different "bin" folder you have to define it in your "composer.json" file (see https://getcomposer.org/doc/articles/vendor-binaries.md#can-vendor-binaries-be-installed-somewhere-other-than-vendor-bin-).

If you want to use the source code directly:
- `git clone https://github.com/rokka-io/rokka-client-php-cli.git`
- `cd rokka-client-php-cli && composer install`
- use the CLI from `bin/rokka-cli`

**Comment by ChristianRiesen** [comment]
A fix was added and tagged as 1.0.1

**Comment by dbu** [comment]
the readme was fixed
